opflexagent/utils/ep_managers/endpoint_file_manager.py

```diff
@@ -507,7 +507,7 @@ def filter_cidr_aaps(aaps):
 
     def _list_to_range(self, vlans_list):
         vlans_list = list(set(vlans_list))
-        vlans_list = filter(lambda a: a > 0 and a < 4094, vlans_list)
+        vlans_list = list(filter(lambda a: a > 0 and a < 4094, vlans_list))
```

Comment:
The guideilnes from https://wiki.openstack.org/wiki/Python3 say:
"Replace filter(lambda obj: test(obj), data) with [obj for obj in data if test(obj)]"

Reply:
Sure, I will change that line.

Reply:
Updated in latest commit